app/src/main/java/org/mozilla/focus/browser/cursor/CursorViewModel.kt

```diff
@@ -74,18 +74,25 @@ class CursorViewModel(
     private var updateLoop: Deferred<Unit>? = null
 
     private fun update(deltaMillis: Long) {
+        /**
+         * Gets the amount velocity should change per frame. The algorithm was determined
+         * experimentally and does not adhere to kinematic math.
+         */
+        fun getDeltaVelPerFrame() = (vel + 1) * DELTA_VEL_CHANGE_PER_FRAME - vel
```

Comment:
I still think it's a little weird that we have this constant for acceleration change over time, but don't use it properly...? anyway, it's fine, just weird.